src/tests/dbus-tests/test_70_encrypted.py

```diff
@@ -449,5 +455,37 @@ def test_resize(self):
         clear_size3 = self.get_block_size(clear_dev)
         self.assertEqual(clear_size3, clear_size)
 
+    def _get_default_luks_version(self):
+        if not os.path.exists(UDISKS_CONFIG_FILE):
+            self.fail('UDisks config file not found.')
+
+        config = configparser.ConfigParser()
+        config.read('/etc/udisks2/udisks2.conf')
```

Comment:
Shouldn't this use the `UDISKS_CONFIG_FILE` constant instead?

Reply:
Fixed.